extensions/sql-database-projects/src/common/constants.ts

```diff
@@ -188,7 +188,7 @@ export const dockerContainerNotRunningErrorMessage = localize('dockerContainerNo
 export const dockerContainerFailedToRunErrorMessage = localize('dockerContainerFailedToRunErrorMessage', "Failed to run the docker container");
 export const connectingToSqlServerOnDockerMessage = localize('connectingToSqlServerOnDockerMessage', "Connecting to SQL Server on Docker");
 export const deployProjectFailedMessage = localize('deployProjectFailedMessage', "Failed to open a connection to the deployed database'");
-export const containerAlreadyExistForProject = localize('containerAlreadyExistForProject', "Other servers on container already exist for the project. Do you want to delete them?");
+export const containerAlreadyExistForProject = localize('containerAlreadyExistForProject', "Other servers on containers already exist for the project. Do you want to delete them?");
```

Comment:
@dzsquared can you help here? I feel like we can improve a bit further.

Reply:
`This project has containers currently running. Do you want to delete them before deploying a new one?`

Maybe we should include something about why you'd want to do this as well?

Reply:
We should close on this @Charles-Gagnon. Any concerns on this string, folks? @llali @kisantia @Benjin

Reply:
Reworded it slightly to `Containers already exist for this project. Do you want to delete them before deploying a new one?`

I opted not to include information about why they'd want to do that since it'd start to get pretty verbose. Happy to make changes now if there are any strong opinions though, otherwise I figure we can at least fix the text now and then revisit the string later if needed.